tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -158,8 +163,172 @@ def test_job_level_scaling(
     _submit_job_full_capacity(scheduler_commands, remote_command_executor)
 
 
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+def test_scaling_special_cases(
+    region, pcluster_config_reader, clusters_factory, scaling_odcr_stack, scheduler_commands_factory, test_datadir
+):
+    full_cluster_size = 1600  # no of nodes after scaling the cluster up
+    downscaled_cluster_size = 1570  # no of nodes after scaling down the cluster
+    max_scaling_time = 7  # Chosen based on running the test multiple times
```

Comment:
Sure, I can add a description of the special cases being tested.

> Can we test the same condition with different values ?

Not really, for the `1570`, the custom partitions refer to nodes up to `1569`. We want to avoid down scaling to a point where the custom partitions refer to nodes that no longer exist. This will result in slurmctld restart failing. I'll add a comment on this since it's relevant.